### DISCUSSION TITLE
myblog/ultimate-linux-arm64-workstation/

### DISCUSSION THREAD

**Discussion by giscus** [jasoneckert/jasoneckert.github.io#7] [Announcements]
# myblog/ultimate-linux-arm64-workstation/

My personal website and blog

https://jasoneckert.github.io/myblog/ultimate-linux-arm64-workstation/

**Comment by elect86** [comment]
Super interesting read

How may I add just the cpu indicator (am I right?) on the top right corner?

**Reply by jasoneckert** [reply]
Ah - that's just part of the [waybar configuration](https://github.com/jasoneckert/sway-dotfiles/tree/main/waybar) (cpu is one of the standard waybar elements, you just have to add your own icon from font-awesome).

**Reply by elect86** [reply]
may I ask you a couple of further questions?

Have you considered jumping on a M2 Ultra? 

How much has been your highest memory usage ever so far? Would have 128GB also been fine?

**Reply by jasoneckert** [reply]
Of course!  No, I haven't considered upgrading from the M1 Ultra to anything else for three reasons:
a) The M1 Ultra is still insanely fast for everything.
b) The M2 Ultra is only slightly faster than the M1 Ultra (probably not even noticeable in practice). 
c) Because it was on the market the longest, the M1 Ultra has the most hardware support from the Asahi team.

I do use a great deal of memory at times - usually around 90GB when running my large BSD VM and K3s staging environment for testing simultaneously.  I think the most I peaked at was just over 100GB (103GB IIRC), and the average when I'm just running my normal server services and other software (personal/professional) is around 30GB. So, 128GB is more than enough for me.

**Comment by RayanJobs** [comment]
Huge issue i see is lack of multiple monitor support...

**Reply by jasoneckert** [reply]
The way I get around that issue is by using a single curved super wide monitor (it was actually cheaper than buying two smaller ones to use together).

**Reply by RayanJobs** [reply]
The thing that sucks is usbc monitor support has been an open issue for over 12 months now. I believe only the hdmi port works. I dont understand why it is so difficult to support a monitor via thunderbolt. I have an M1 Ultra
